WooCommerce/Classes/ViewRelated/Orders/Order Details/Order Custom Fields/OrderCustomFieldsDetails.swift

```diff
@@ -0,0 +1,63 @@
+import SwiftUI
+
+struct OrderCustomFieldsDetails: View {
+    @Environment(\.presentationMode) var presentationMode
+
+    let customFields: [OrderCustomFieldsViewModel]
+
+    var body: some View {
+        NavigationView {
+            GeometryReader { geometry in
+                ScrollView {
+
+                    Color(.listBackground).edgesIgnoringSafeArea(.all)
+
+                    VStack(alignment: .leading) {
+                        ForEach(customFields) { customField in
+                            TitleAndSubtitleRow(
+                                title: customField.title,
+                                subtitle: customField.content
+                            )
+                            Divider()
+                                .padding(.leading)
+                        }
+                    }
+                    .background(Color(.basicBackground))
```

Comment:
nit: I'd suggest using our `.listForeground` color here:

```suggestion
                    .background(Color(.listForeground))
```

It ends up with the same color, but it's more semantically accurate and will help if we ever update our list style.

Reply:
Sure! Fixed here: https://github.com/woocommerce/woocommerce-ios/pull/7257/commits/9decc6ef0b883c2a0254b87f0a3c14f883f1a93b